CNCF/general_technical_review.md

```diff
@@ -0,0 +1,459 @@
+
+    * Describe the project defaults  
+
+KubeVirt provides configuration defaults, allowing users to “just” deploy KubeVirt without requiring any input.
+
+    * Outline any additional configurations from default to make reasonable use of the project  
+KubeVirt has one default configuration that it shares between KubeVirt components.
+
+    * Describe any new or changed API types and calls \- including to cloud providers \- that will result from this project being enabled and used  
+
+Most KubeVirt APIs are Kubernetes API extensions (CRDs). They are documented at http://kubevirt.io/api-reference/
+
+    * Describe compatibility of any new or changed APIs with API servers, including the Kubernetes API server   
+
+Compatibility is provided in https://github.com/kubevirt/kubevirt/blob/main/docs/kubernetes-compatibility.md
+
+    * Describe versioning of any new or changed APIs, including how breaking changes are handled  
+For releases KubeVirt follows Semantic Versioning - just like Kubernetes does.
+For CRDs KubeVirt follows Kubernetes standards for API versioning to ensure stability, compatibility, and a clear evolution of its APIs.
+KubeVirt APIs are versioned with labels such as v1alpha1, v1beta1, and v1 - where Alpha is considered experimental, beta is more stable with some backward compatibility and v1 is fully mature APIs with strong backward compatibility guarantees.
+KubeVirt v1 APIs maintain backward compatibility and changes that are not backward compatible are avoided in stable APIs.
+Deprecation policy
+
+  * Describe the project’s release processes, including major, minor and patch releases.
+
+The release process is described in https://github.com/kubevirt/kubevirt/blob/main/docs/release.md
+
+
+### Installation
+
+  * Describe how the project is installed and initialized, e.g. a minimal install with a few lines of code or does it require more complex integration and configuration?  
+
+KubeVirt is providing an Operator for deployment and application life-cycle management. Alternatively, deployment can be manually performed by installing key components from the kubevirt and containerized-data-importer repositories. These are all installed using `kubectl`.
+
+There are then additional optional network plugins that can be installed.
+
+Reference: https://kubevirt.io/user-guide/cluster_admin/installation/#installing-kubevirt-on-kubernetes
+
+  * How does an adopter test and validate the installation?
+
+An adopter can run the KubeVirt testsuite on their deployment.
+
+### Security
+
+  * Please provide a link to the project’s cloud native [security self assessment](https://github.com/cncf/tag-security/blob/main/community/assessments/guide/self-assessment.md).  
+
+The self assessment is in progress and tracked at https://github.com/kubevirt/community/issues/335.
+This document will be updated pointing to the self-assessment once it is completed.
+
+    * How are you satisfying the tenets of cloud native security projects?
```

Comment:
```suggestion
  * How are you satisfying the tenets of cloud native security projects?
```
And more below

Reply:
Yeah, that's just a mess.

This is what their original [doc](https://raw.githubusercontent.com/cncf/toc/refs/heads/main/tags/resources/toc-supporting-guides/general-technical-questions.md) looks like.

We have to answer each bullet, but it just creates a mess, since some of the bullets are nested.